README.md

```diff
@@ -1198,6 +1198,10 @@ Tells if the application is running in an emulator.
 const isEmulator = DeviceInfo.isEmulator(); // false
 ```
 
+**Android Permissions**
+
+* [android.permission.READ_PHONE_STATE](https://developer.android.com/reference/android/Manifest.permission.html#READ_PHONE_STATE)
```

Comment:
I add this permission to check if is not a Emulator, this permission has been used at the class RNDeviceModule.java in methods `checkOperatorNameAndroid() `and `checkImsi()`.

Reply:
You might encourage developers to add it to their src/debug/AndroidManifest.xml files - that's a good spot for things like this

Reply:
Okay, I'll change it